hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupBase.java

```diff
@@ -337,7 +337,7 @@ public static void setUpHelper() throws Exception {
     if (useSecondCluster) {
       BACKUP_REMOTE_ROOT_DIR = new Path(
         new Path(TEST_UTIL2.getConfiguration().get("fs.defaultFS")) + BACKUP_REMOTE_ROOT_DIR)
-          .toString();
+        .toString();
```

Comment:
Is this a formatting change that we want?

Reply:
This is just done by running 'mvn spotless:apply' after changing the spotless version. If we do not apply this, the spotless check in pre commit will fail...

Reply:
I understand. My question/concern is more about the change in configuration that results in this change to source formatting. I'm asking if the configuration change desirable.

Reply:
This is not intentional but every time we update the version of spotless, there will be some small changes on the formatting result. I guess when updating spotless version, the eclipse formatter version will be updated, which may cause some small style changes.

Reply:
I see. That's annoying.

Thanks Duo!